keyboards/electronlab/klor/keyboard.json

```diff
@@ -0,0 +1,372 @@
+            {"matrix": [1, 2], "x": 30, "y": 20, "flags": 4},
+            {"matrix": [1, 3], "x": 46, "y": 12, "flags": 4},
+            {"matrix": [1, 4], "x": 60, "y": 19, "flags": 4},
+            {"matrix": [1, 5], "x": 75, "y": 21, "flags": 4},
+            {"matrix": [5, 5], "x": 149, "y": 21, "flags": 4},
+            {"matrix": [5, 4], "x": 163, "y": 19, "flags": 4},
+            {"matrix": [5, 3], "x": 178, "y": 12, "flags": 4},
+            {"matrix": [5, 2], "x": 194, "y": 20, "flags": 4},
+            {"matrix": [5, 1], "x": 209, "y": 30, "flags": 4},
+            {"matrix": [5, 0], "x": 224, "y": 28, "flags": 1},
+            {"matrix": [2, 0], "x": 3, "y": 41, "flags": 1},
+            {"matrix": [2, 1], "x": 17, "y": 42, "flags": 4},
+            {"matrix": [2, 2], "x": 31, "y": 33, "flags": 4},
+            {"matrix": [2, 3], "x": 46, "y": 25, "flags": 4},
+            {"matrix": [2, 4], "x": 60, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 75, "y": 34, "flags": 4},
+            {"matrix": [6, 5], "x": 149, "y": 34, "flags": 4},
+            {"matrix": [6, 4], "x": 163, "y": 32, "flags": 4},
+            {"matrix": [6, 3], "x": 178, "y": 25, "flags": 4},
+            {"matrix": [6, 2], "x": 193, "y": 33, "flags": 4},
+            {"matrix": [6, 1], "x": 211, "y": 17, "flags": 4},
+            {"matrix": [6, 0], "x": 221, "y": 41, "flags": 1},
+            {"matrix": [3, 1], "x": 58, "y": 48, "flags": 1},
+            {"matrix": [3, 2], "x": 74, "y": 49, "flags": 1},
+            {"matrix": [3, 3], "x": 89, "y": 54, "flags": 1},
+            {"matrix": [3, 4], "x": 102, "y": 63, "flags": 1},
+            {"matrix": [7, 4], "x": 122, "y": 63, "flags": 1},
+            {"matrix": [7, 3], "x": 135, "y": 54, "flags": 1},
+            {"matrix": [7, 2], "x": 150, "y": 49, "flags": 1},
+            {"matrix": [7, 1], "x": 166, "y": 48, "flags": 1}
+        ],
+        "split_count": [21, 21],
+        "led_process_limit": 10,
+        "sleep": true,
+        "timeout": 120000,
+        "hue_steps": 3
+    },
+    "split": {
+        "enabled": true,
+        "bootmagic": {
+            "matrix": [4, 1]
+        },
+        "serial": {
+            "driver": "vendor",
+            "pin": "GP1"
+        },
+        "encoder": {
+            "right": {
+                "rotary": [
+                    {"pin_a": "GP29", "pin_b": "GP28", "resolution": 2}
+                ]
+            }
+        },
+        "transport": {
+            "sync": {
+                "activity": true,
+                "detected_os": true,
+                "haptic": true,
+                "indicators": true,
+                "layer_state": true,
+                "matrix_state": true,
+                "modifiers": true,
+                "oled": true
+            }
+        },
+    },
+    "url": "https://github.com/Lefuneste83/KLOR",
+    "usb": {
+        "device_version": "1.4.0",
+        "pid": "0x0001",
+        "vid": "0x3A3C"
```

Comment:
While this isn't currently assigned, it will be soon.  The further part 0x4000, the better.

Reply:
The comment "The further part 0x4000, the better" is not very clear to me. Can you be more specific?

Reply:
Vendor IDs are assigned by USB IF (the ***the*** USB regulatory body), and they hand out the vendor IDs sequentially.  While this isn't in a block that is being assigned yet, 0x38xx are being actively assigned. 

So, the higher the number, the better.